kernel/src/main/java/org/kframework/kast/KastFrontEnd.java

```diff
@@ -112,7 +116,10 @@ public int run() {
                 options.module = def.mainSyntaxModuleName();
                 switch (options.input) {
                     case KORE:
-                        unparsingMod = def.languageParsingModule();
+                        Module languageParsingModule = def.languageParsingModule();
+                        Module k = def.kompiledDefinition.getModule("K").get();
+                        unparsingMod = Constructors.Module("K-LANGUAGE-PARSING",
+                                Set(languageParsingModule, k) , Set(), KORE.Att());
```

Comment:
What's going on here?

Reply:
Yeah, this seems strange.

Reply:
Well, the `#KRewrite` label is defined in the `K` module, which is used for parsing rules and not being computed for the compiled definition. 

I'm just creating a module importing both the k module and the parsing module here on-spot, to make `#KRewrite` available to the definition.

Do you have any suggestions?

Reply:
Just create a new module in kast.k containing just the rewrite production and import it in both K and K-TERM.

Reply:
done. please take another look.